FCCee/IDEA/compact/IDEA_o1_v03/FiberDualReadoutCalo_o1_v01.xml

```diff
@@ -0,0 +1,698 @@
+      <composite n="2" ref="F"/>
+      <composite n="2" ref="H"/>
+      <property name="RINDEX" ref="RI_FluoPoly"/>
+    </material>
+    <material name="PMMA">
+      <D value="1.19" unit="g/cm3"/>
+      <composite n="5" ref="C"/>
+      <composite n="8" ref="H"/>
+      <composite n="2" ref="O"/>
+      <property name="RINDEX" ref="RI_PMMA"/>
+      <property name="ABSLENGTH" ref="AbsLen_PMMA"/>
+    </material>
+    <material name="PolyvinylChloride">
+      <D value="1.3" unit="g/cm3"/>
+      <fraction n="0.04838" ref="H"/>
+      <fraction n="0.38436" ref="C"/>
+      <fraction n="0.56726" ref="Cl"/>
+    </material>
+
+    <material name="DR_Air">
+      <D type="density" unit="g/cm3" value="0.0012"/>
+      <fraction n="0.754" ref="N"/>
+      <fraction n="0.234" ref="O"/>
+      <fraction n="0.012" ref="Ar"/>
+      <property name="RINDEX" ref="RI_Air"/>
+    </material>
+    <material name="DR_Polystyrene">
+      <D value="1.032" unit="g/cm3"/>
+      <composite n="19" ref="C"/>
+      <composite n="21" ref="H"/>
+      <property name="RINDEX" ref="RI_PS"/>
+      <property name="ABSLENGTH" ref="AbsLen_PS"/>
+      <property name="SCINTILLATIONCOMPONENT1" ref="scintFast_PS"/>
+      <constant name="SCINTILLATIONYIELD" value="13.9/keV"/>
+      <constant name="SCINTILLATIONTIMECONSTANT1" value="2.8*ns"/>
+      <constant name="RESOLUTIONSCALE" value="1."/>
+      <constant name="BirksConstant" value="0.126*mm/MeV"/> <!-- Add Birks const -->
+    </material>
+    <material name="DR_PyrexGlass">
+      <D type="density" value="2.23" unit="g/cm3"/>
+      <fraction n="0.806" ref="SiliconOxide"/>
+      <fraction n="0.130" ref="BoronOxide"/>
+      <fraction n="0.040" ref="SodiumOxide"/>
+      <fraction n="0.023" ref="AluminumOxide"/>
+      <property name="RINDEX" ref="RI_Glass"/>
+      <property name="ABSLENGTH" ref="Abslength_Glass"/>
+    </material>
+  </materials>
+
+  <surfaces>
+    <opticalsurface name="/world/DRcalo#SiPMSurf" finish="polished" model="glisur" type="dielectric_metal">
+      <property name="REFLECTIVITY" ref="refl_SiPM"/>
+      <property name="EFFICIENCY" ref="eff_SiPM"/>
+    </opticalsurface>
+    <opticalsurface name="/world/DRcalo#FilterSurf" finish="polished" model="glisur" type="dielectric_dielectric">
+      <property name="REFLECTIVITY" ref="filterRef"/>
+      <property name="TRANSMITTANCE" ref="filterEff"/>
+    </opticalsurface>
+    <opticalsurface name="/world/DRcalo#MirrorSurf" finish="polished" model="glisur" type="dielectric_metal">
+      <property name="REFLECTIVITY" ref="mirrorRef"/>
+      <property name="TRANSMITTANCE" ref="mirrorEff"/>
+    </opticalsurface>
+  </surfaces>
+
+  <!-- this is where we actually define our detector: -->
+  <detectors>
+    <detector id="DetID_FiberDRCalo" name="DRcalo" type="FiberDualReadoutCalo_o1_v01" readout="DRcaloSiPMreadout" region="FastSimOpFiberRegion" reflect="true" vis="DRCInvisible">
+      <sensitive type="DRcaloSiPMSD"/>
```

Comment:
seems like CI is failing because we have a customized SD (type `DRcaloSiPMSD`) but the CI still looks for the standard SD (type `calorimeter`). Do we want to update the CI matrix itself here? (since the usage of the standard SD is not intended)
```
DDSim            INFO Setting up SD for DRcalo with No Default
DDSim            ERROR Cannot find Action for detector DRcalo. You have to extend "action.mapAction"
DDSim            ERROR Failed setting up sensitive detector Cannot find Action
```

Reply:
Don't you just have to provide a steering file to configure the correct SD?
(is that what you mean by updating the CI matrix?)

Reply:
That's exactly what we do when running 'offline' (and it works fine), but I suspect the github CI simply executes `ddsim --compactFile IDEA_o1_v03.xml` (maybe this [one](https://github.com/key4hep/k4geo/blob/main/test/CMakeLists.txt#L94)?) without a steering file and then fails. So I was wondering whether we want to fix this.

Reply:
The test must not fail, so we want to fix this.

Reply:
I think I figured out how to do this without pushing a steering file. @swkim95 can you try the following for this [line](https://github.com/key4hep/k4geo/blob/main/test/CMakeLists.txt#L94)? `ddsim --compactFile=${CMAKE_CURRENT_SOURCE_DIR}/../FCCee/IDEA/compact/IDEA_o1_v03/IDEA_o1_v03.xml --runType=batch -G -N=1 --outputFile=testIDEA_o1_v03.slcio --action.calorimeterSDTypes='calorimeter DRcaloSiPMSD' --action.mapActions='DRcalo DRCaloSDAction' )`